packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -8,7 +8,17 @@ import {
 } from '@blueprintjs/core';
 import { MenuItem2, Popover2 } from '@blueprintjs/popover2';
 import { subject } from '@casl/ability';
-import { IconDots, IconPencil } from '@tabler/icons-react';
+import {
```

Comment:
we could also use                                 <MantineIcon
 which is a wrapper with a theme, this is normally used when we need to apply a different color or something. 
 
 But using the icon directly works here too